<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowTetris</class>
 <widget class="QMainWindow" name="MainWindowTetris">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>910</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>910</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>910</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Snake</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>misc/images/tetris-game.png</normaloff>misc/images/tetris-game.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>403</width>
      <height>880</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>403</width>
      <height>880</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>403</width>
      <height>880</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>403</width>
      <height>880</height>
     </size>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="nextTetromino">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>163</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="Score">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Score:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="BestScore">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Best score:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>790</y>
      <width>161</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="EndGame">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>End - Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Stop">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stop - P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="start">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start - Space</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
